script/foreman-debug

```diff
@@ -19,6 +19,8 @@ This program can be used on Foreman instances, Smart Proxy instances or
 backend services separately.
 
 OPTIONS:
+  -E      Detailed output incl. host names or domains
```

Comment:
This is misleading, it's the contents of the database and implies that the hostname and domain will be obscured otherwise.